Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -124,11 +111,6 @@ class MessagingInAppImplementationTest: UnitTest {
     func test_eventListeners_expectCallListenerForEachEvent() {
         let givenGistMessage = Message.random
 
-        backgroundQueueMock.addTrackInAppDeliveryTaskReturnValue = (
-            success: true,
-            queueStatus: QueueStatus.successAddingSingleTask
-        )
-
```

Comment:
[Reason](https://github.com/customerio/customerio-ios/pull/461/files#r1441777027)

Reply:
instead of removing, we need to test/comment to check if this is being sent to `EventBus` mock

Reply:
Since `EventBus` work is still in progress, I am adding a `FIXME: [CDP]` note to this to make the changes later. `FIXME: [CDP]` because it is consistent with other pending CDP tweaks we might require to make before releasing.